cypress/e2e/misdemeanor/fishing/pages/m-offense-fish-2-cont.cy.ts

```diff
@@ -0,0 +1,27 @@
+describe('m-offense-fish-2-cont', () => {
+  it('have all necessary information', () => {
+    cy.visit('/calculator/m-offense-fish-2-cont');
+
+    cy.get('strong:visible')
+      .contains('Are you a member of a tribe?')
+      .should('have.length', 1);
+    cy.get('p:visible')
+      .contains('You can also apply on behalf of a deceased family member in the tribe who meets the criteria.')
+      .should('have.length', 1);
+  });
+
+  it('when select NO option, open continue to general criteria', () => {
+    cy.visit('/calculator/m-offense-fish-2-cont');
+
+    cy.get('a:visible').contains(/^No$/).click();
+    cy.url().should('include', '/calculator/m-offense-fish-6-cont');
+  });
+
+  it('when select "I\'m not sure" option, open pop-up with some information', () => {
```

Comment:
I see this test on every page, is it necessary to test on every page when this question is always going to be the exact same thing on every page?

Reply:
Thats is a good question lol.
As calculator its just a page with changes by `slug`, i think we can just check this in one page and we kinda ensure that it will be working in all other pages.
I think its better to remove those duplications